frontend/src/metabase/common/components/EntityPicker/types.ts

```diff
@@ -49,3 +70,7 @@ export type ListProps<
 };
 
 export type FilterItemsInPersonalCollection = "only" | "exclude";
+
+export type TabFolderState = Partial<
+  Record<EntityPickerTabId, TypeWithModel<unknown, string>>
```

Comment:
This `TypeWithModel<unknown, string>` is likely to cause problems later. I will tackle it when it happens.